go.mod

```diff
@@ -21,41 +21,41 @@ require (
 	github.com/fsnotify/fsnotify v1.5.4
 	github.com/ghodss/yaml v1.0.0
 	github.com/golang/glog v1.0.0
-	github.com/golangci/golangci-lint v1.46.2
+	github.com/golangci/golangci-lint v1.49.0
 	github.com/google/go-cmp v0.5.8
 	github.com/google/renameio v0.1.0
 	github.com/imdario/mergo v0.3.13
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v0.0.0-20220824134416-0f86a223d4bc
+	github.com/openshift/api v0.0.0-20220915134421-1265e9925688
 	github.com/openshift/client-go v0.0.0-20220525160904-9e1acff93e4a
```

Comment:
shouldn't we also bump other openshift libs as well like client-go, library-go  to latest?

Reply:
Fine by me, will do.